### ISSUE TITLE
DataStore: ClassCastException when setting limit on a stream - cannot cast to HasComparableOperationsIndex

### ISSUE THREAD

**Issue by darzynkiewicz** [speedment/speedment#567]
This happens on version 1.1.14 when I map String column using Y/N to Boolean mapper. See screenshot.
```
Caused by: com.syncron.bpp.common.exception.transfer.ServiceException: [INTERNAL] java.lang.ClassCastException: com.speedment.enterprise.datastore.runtime.internal.fieldcache.IndexedBooleanFieldCache cannot be cast to com.speedment.enterprise.datastore.runtime.internal.fieldcache.trait.indexed.HasComparableOperationsIndex
	at com.speedment.enterprise.datastore.runtime.internal.stream.InitialCacheStream.comparableHelper(InitialCacheStream.java:881) [datastore-runtime-1.1.14.jar:]
	at com.speedment.enterprise.datastore.runtime.internal.stream.InitialCacheStream.findHelper(InitialCacheStream.java:616) [datastore-runtime-1.1.14.jar:]
	at com.speedment.enterprise.datastore.runtime.internal.stream.InitialCacheStream.applyConfig(InitialCacheStream.java:477) [datastore-runtime-1.1.14.jar:]
	at com.speedment.enterprise.datastore.runtime.internal.stream.InitialCacheStream.limit(InitialCacheStream.java:171) [datastore-runtime-1.1.14.jar:]
```
![yntobooleanmapping](https://user-images.githubusercontent.com/33186073/32931041-ea9b7782-cb16-11e7-89eb-9f0758b25154.png)

**Comment by minborg** [comment]
Enterprise Function

**Comment by lawesson** [comment]
Fixed in Enterprise, perhaps the commit messade reference to the issue number does not work since the issue is reported in open source but fixed in enterprise?

a2e3cd5f0f9594ecb3d31867303b33fd64726269

**Comment by lawesson** [comment]
Fix will be released in speedment-enterprise 1.1.15 (and available in custom built snapshots prior to that).
